### PR TITLE
Cleanup of the test package.

### PR DESCRIPTION
**Pull request by Donnerbart**
The normal, boring cleanup stuff, driven by IntelliJ and our current CheckStyle rules (which are not automatically applied here).

### CODE REVIEW

**Comment by Donnerbart** [comment]
verify

File: hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java [hunk 1]
```diff
@@ -76,17 +78,16 @@ public static String getTestMethodName() {
     }
 
     /**
-     * Creates a BlockJUnit4ClassRunner to run {@code klass}
+     * Creates a BlockJUnit4ClassRunner to run {@code classType}
      *
      * @throws org.junit.runners.model.InitializationError if the test class is malformed.
      */
-    public AbstractHazelcastClassRunner(Class<?> klass) throws InitializationError {
-        super(klass);
```
Comment (pveentjer) [inline hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java]: classType is not an improvements over klass. With klass it is at at least obvious that a 'class' is expected, but that the name is in use. classType is going to be the type of a class? So either use klass or clazz..

File: hazelcast/src/test/java/com/hazelcast/test/TestHazelcastInstanceFactory.java [hunk 2]
```diff
@@ -149,10 +151,10 @@ public void terminateAll() {
     private static List<Address> createAddresses(AtomicInteger ports, String... addressArray) {
         checkElementsNotNull(addressArray);
 
-        final int count = addressArray.length;
+        int count = addressArray.length;
         List<Address> addresses = new ArrayList<Address>(count);
-        for (int i = 0; i < count; i++) {
-            addresses.add(createAddress(addressArray[i], ports.incrementAndGet()));
+        for (String anAddressArray : addressArray) {
```
Comment (pveentjer) [inline hazelcast/src/test/java/com/hazelcast/test/TestHazelcastInstanceFactory.java]: anAddressArray to indicate a field of type string? I think it should be 'address'.

File: hazelcast/src/test/java/com/hazelcast/test/TestPartitionUtils.java [hunk 3]
```diff
@@ -170,17 +161,15 @@ public int getPartitionId() {
 
         @Override
         public void run() {
-            final InternalPartitionService partitionService = node.nodeEngine.getPartitionService();
-            final long[] replicaVersions = partitionService.getPartitionReplicaVersions(partitionId);
-            this.replicaVersions = Arrays.copyOf(replicaVersions, replicaVersions.length);
+            InternalPartitionService partitionService = node.nodeEngine.getPartitionService();
+            long[] replicaVersionsArray = partitionService.getPartitionReplicaVersions(partitionId);
```
Comment (pveentjer) [inline hazelcast/src/test/java/com/hazelcast/test/TestPartitionUtils.java]: I prefer the replicaVersions. Array in this case only adds noise; doesn't provide any added value.

**Comment by pveentjer** [comment]
Minor but important comments. Once resolved I'm happy to put my thumbs up sign on it.

File: hazelcast/src/test/java/com/hazelcast/test/TestPartitionUtils.java (hunk 3 above)
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/test/TestPartitionUtils.java]: `replicaVersions` is already the name of a field. So in the old code `final long[] replicaVersions` was hiding this field, so a `this.replicaVersions` was needed a line below. Hiding fields seems not to be a good practice for me (it's also a SonarQube issue).

File: hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java (hunk 1 above)
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java]: I just wanted to give the names a unified and more English name. I think I saw `classType` in the JDK itself, but I may be wrong. I picked `clazz` for now.

Comment (pveentjer) [inline hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java]: With type of class you start to enter complex type systems. Where a type of a type (class) is also a 'kind'. So it puts me on the wrong foot.

**Comment by pveentjer** [comment]
:+1:

**Pull request merged**